components/brave_ads/content/browser/search_result_ad/search_result_ad_parsing_unittest.cc

```diff
@@ -75,7 +75,7 @@ class TestWebPageConstructor final {
   }
 
   schema_org::mojom::EntityPtr CreateCreativeEntity() {
-    const char* search_result_ad_string_attributes[] = {
+    const char* kSearchResultAdStringAttributes[] = {
```

Comment:
this should be: `static constexpr const char*`

Reply:
Fixed